gnd/src/main/java/com/google/android/gnd/ui/home/featuredetails/FeatureDetailsViewModel.java

```diff
@@ -82,6 +93,25 @@ public FeatureDetailsViewModel(
                 featureMutations, observationMutations, (f, o) -> !f.isEmpty() && !o.isEmpty()));
   }
 
+  /** Returns true if the user is {@link Role#OWNER} or {@link Role#MANAGER} of the project. */
+  private boolean isActiveUserAuthorizedToModifyFeature(Feature feature) {
+    Role role = userRepository.getUserRole(feature.getProject());
+    return role == Role.OWNER || role == Role.MANAGER;
```

Comment:
They can also modify if they're the owner of the feature. Do we have that in Feature yet?

Reply:
It is present under `created`. Done